src/transport_picoquic.cpp

```diff
@@ -1814,6 +1834,8 @@ void PicoQuicTransport::mark_stream_active(const TransportConnId conn_id, const
         return;
     }
 
+    data_ctx_it->second.mark_stream_active = false;
```

Comment:
So it's okay to set this to false even if we didn't mark as active because the stream ID = 0?

Reply:
Yeah, the mark active flag being true prevents duplicate requests to mark active. In this case, we need the next enqueue to mark active again because here we ignore/discard.